EESSI-install-software.sh

```diff
@@ -232,7 +237,11 @@ pr_diff=$(ls [0-9]*.diff | head -1)
 # install any additional required scripts
 # order is important: these are needed to install a full CUDA SDK in host_injections
 # for now, this just reinstalls all scripts. Note the most elegant, but works
-${TOPDIR}/install_scripts.sh --prefix ${EESSI_PREFIX}
+
+# Only run install_scripts.sh if not dev.eessi.io for security
+if [[ "${EESSI_CVMFS_REPO}" != /cvmfs/dev.eessi.io ]]; then
```

Comment:
Doing this makes sense, but it will cause fallout below where the `install_cuda_host_injections.sh` script is called.

OK to ignore for now since it's only relevant when we're building CUDA software in `dev.eessi.io`), but we'll probably need to have a construct like this:

```
if [[ "${EESSI_CVMFS_REPO}" == /cvmfs/dev.eessi.io ]]; then
    SCRIPTS_DIR=/cvmfs/software.eessi.io
else
    SCRIPTS_DIR=${EESSI_PREFIX}
    ${TOPDIR}/install_scripts.sh --prefix ${EESSI_PREFIX}
fi
```

and then use `$SCRIPTS_DIR` when calling `install_cuda_host_injections.sh` & co below

Reply:
Make this a follow up PR